src/main/java/org/jenkinsci/plugins/workflow/cps/steps/LoadStepExecution.java

```diff
@@ -54,7 +54,7 @@ public boolean start() throws Exception {
 
         // execute body as another thread that shares the same head as this thread
         // as the body can pause.
-        cps.newBodyInvoker(t.getGroup().export(script))
+        cps.newBodyInvoker(t.getGroup().export(script), true)
```

Comment:
The reported issue was about `parallel`, but while looking for unmatched calls to `export` I found this one too.